CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+## 15.5.4
+* Fix UMD Build. [See this commit](https://github.com/reactjs/prop-types/commit/31e9344ca3233159928da66295da17dad82db1a8).
+* Remove dependencies on React in `package.json`. [See this commit](https://github.com/reactjs/prop-types/commit/158198fd6c468a3f6f742e0e355e622b3914048a).
```

Comment:
This wasn't removing a dependency. Just fixing a bad package URL. (Wouldn't affect installs.)